packages/preact-iso/router.d.ts

```diff
@@ -4,7 +4,9 @@ export const LocationProvider: FunctionComponent;
 
 export function Router(props: { onLoadEnd?: () => void, onLoadStart?: () => void, children?: VNode[] }): VNode;
 
-export const useLoc: () => { url: string, path: string, query: Record<string, string>, route: (url: string) => void };
+type LocationHook = () => { url: string, path: string, query: Record<string, string>, route: (url: string) => void };
+export const useLoc: LocationHook;
+export const useLocation: LocationHook;
```

Comment:
This could also simply be `... useLocation: typeof useLoc;` and we could get rid of the extra type definiton. I have no real opinions on which is nicer.